public/app/percona/settings/components/PlatformLogin/PlatformLogin.constants.ts

```diff
@@ -1,3 +1,8 @@
 export const TERMS_OF_SERVICE_URL = 'https://per.co.na/pmm/platform-terms';
 
 export const PRIVACY_POLICY_URL = 'https://per.co.na/pmm/platform-privacy';
+
+export const RESET_PASSWORD_URL =
+  process.env.NODE_ENV !== 'production'
```

Comment:
Isn't it the opposite? 
```suggestion
  process.env.NODE_ENV === 'production'
```